packages/pagination/src/PaginationControls.tsx

```diff
@@ -153,9 +153,9 @@ const PaginationControls = ({
         ''
       )}
       {showPaginationText && (
-        <div data-testid="pagination-text" className="pagination-text pt-1 pl-2 pr-2">
+        <li data-testid="pagination-text" className="pagination-text pt-1 pl-2 pr-2">
```

Comment:
I'm wondering if we should move this outside the `Pagination` component and wrap them both with a fragment. I'm not sure if it will mess with the styles/layout though. I'll have to check

Reply:
I can play with it really quickly.

Reply:
Yeah it throws the styling off, which is not impossible to fix. But it seems a bit cleaner to keep this under the <ul> as a <li> if you are cool with it.

Reply:
I am cool with it assuming it's compliant